File: web-development/helsinki-mooc/java/ex76/Menu.java
import java.util.ArrayList;

public class Menu {

  //vars
  private ArrayList<String> meals = new ArrayList<String>();

  //constructors
  public Menu() {
    this.meals = meals;
  }

  //methods
  public void add_meal(String meal) {
    this.meals.add(meal);
  }

  public void print_menu() {
    for(String n : meals) {
      System.out.println(n);
    }
  }

}

File: compsci/programming/exersices/helsinki-moocs/python/ex76/Menu.py
class Menu:

    #constructor
    def __init__(self):
        self.meals = []

    #behaviors
    def add_meal(self, meal):
        if meal not in self.meals:
            self.meals.append(meal)
        else:
            print('This meal already exists')

    def print_menu(self):
        for meal in self.meals:
            print(meal)
